src/d3d11/d3d11_context_ext.h

```diff
@@ -56,11 +56,23 @@ namespace dxvk {
     
     void STDMETHODCALLTYPE SetBarrierControl(
             UINT                    ControlFlags);
-    
+
+    bool STDMETHODCALLTYPE LaunchCubinShaderNVX(
```

Comment:
for interfaces like this, do `GridX` as msft style if youre going to use pParams and stuff